js/index.js

```diff
@@ -0,0 +1,17 @@
+// Adding Name, year and copyright in the Footer
+const today = new Date();
+const thisYear = today.getFullYear();
+const footer = document.querySelector("footer");
+const copyright = document.createElement("p");
+copyright.innerHTML = `&copy; Made with &hearts; by Elena Sokolovskaya ${thisYear}`;
+footer.appendChild(copyright);
+
+// Adding skills
+const skills = ["JavaScript", "HTML", "CSS", "GIT"];
+const skillsSection = document.getElementById("Skills");
+const skillsList = skillsSection.querySelector("ul");
+for (let i = 0; i < skills.length; i++) {
```

Comment:
Slightly simpler to use for (let skill of skills) { ... }

Reply:
I didn't fully understand for ... of loop, so I decided to write as I understand it at this stage.